reactive/puppet.py

```diff
@@ -50,26 +36,19 @@ def install_puppet():
 @when_not('puppet.available')
 def puppet_ready():
     """
-    Set status so users know puppet is ready. Set the 'puppet.available' state
-    so other layers can gate puppet operations.
+    Set the `puppet.available` state so other layers can gate
+    puppet operations.
     """
-    hookenv.status_set('active', 'puppet is installed')
```

Comment:
By removing this line this layer will never become active. Meaning that if we charm build this layer and deploy we will be stuck into "maintenance - Installing puppet" state. I guess this is fine when we build on this layer.